.github/workflows/validate-json.yml

```
@@ -0,0 +1,14 @@
+name: Validate JSONs
+
+on: [pull_request]
+
+jobs:
+  verify-json-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Validate JSON
+        uses: docker://orrosenblatt/validate-json-action:latest
+        env:
+          INPUT_JSONS: /disclose/program-list/program-list.json
+          INPUT_SCHEMA: /disclose/program-list/program-list-schema.json
```

```suggestion
          INPUT_JSONS: /program-list/program-list.json
          INPUT_SCHEMA: /program-list/program-list-schema.json
```

that /disclose repository is root so maybe this will work?

---

I had the same thought and attempted this with #c9deb9c unsuccessfully, unfortunately

---

ah oops, didn't see that until just now. sorry. It seems having it didn't necessarily help either. Don't have time to dig in more right now but maybe later this week if no one else gets to it before.

---

Some problem with ajv i think

---

I tried this https://github.com/epoberezkin/ajv/issues/472 
But not working